provision/etc/Makefile.am

```diff
@@ -13,7 +13,7 @@ endif
 
 httpdconfdir = $(httpdir)
 
-dist_conf_DATA = provision.conf livesync.conf dhcpd-template.conf
+dist_conf_DATA = provision.conf livesync.conf dhcpd-template.conf dnsmasq-template.conf
```

Comment:
Additionally, I think we should come up with a cleaner way than installing both dhcpd-template.conf and dnsmasq-template.conf into /etc/warewulf. Having both to me seems like it will lead to a confusing user experience. Ideas?

Reply:
In an ideal world users should never have to edit those templates, so /usr/share/warewulf/ may be a better location.

Reply:
Hmm. Suppose for now let's just stick them both in /etc/warewulf so we're not changing behavior for ISC.